doc/source/whatsnew/v1.2.0.rst

```diff
@@ -523,6 +523,7 @@ Categorical
 - :meth:`Categorical.fillna` will always return a copy, validate a passed fill value regardless of whether there are any NAs to fill, and disallow an ``NaT`` as a fill value for numeric categories (:issue:`36530`)
 - Bug in :meth:`Categorical.__setitem__` that incorrectly raised when trying to set a tuple value (:issue:`20439`)
 - Bug in :meth:`CategoricalIndex.equals` incorrectly casting non-category entries to ``np.nan`` (:issue:`37667`)
+- Bug in :meth:`Categorical.unique`, where the dtype changed in the unique array if there were unused categories in the original array (:issue:`38140`).
```

Comment:
i would make a sub-section abou tthis, showing before & after; this is a non-trivial change to absorb.

Reply:
I agree, we can move it to the "notable bug fixes" section, with an example and showing how you can get back at the old behaviour.